File: Python Project/Ogrenci.py
class Ogrenci:
    def _init_(self, ad, soyad, numara, bolum, fakultesi, giris_yili, sinif):
        self.__ad = ad
        self.__soyad = soyad
        self.__numara = numara
        self.__bolum = bolum
        self.__fakultesi = fakultesi
        self.__giris_yili = giris_yili
        self.__sinif = sinif

    @property
    def ad(self):
        return self.__ad

    @property
    def soyad(self):
        return self.__soyad

    @property
    def numara(self):
        return self.__numara

    @property
    def bolum(self):
        return self.__bolum

    @property
    def fakultesi(self):
        return self.__fakultesi

    @property
    def giris_yili(self):
        return self.__giris_yili

    @property
    def sinif(self):
        return self.__sinif

    def get_ad_soyad(self):
        return f"{self.ad} {self.soyad}"

File: Java Project/Ogrenci.java
public class Ogrenci {

    private final String ad;
    private final String soyad;
    private final String bolum;
    private final String fakulte;
    private final int numara;
    private final int girisYili;
    private final int sinif;

    public Ogrenci(String ad, String soyad, int no, String bolum, String fakulte, int girisYili, int sinif) {
        this.ad = ad;
        this.soyad = soyad;
        this.numara = no;
        this.bolum = bolum;
        this.fakulte = fakulte;
        this.girisYili = girisYili;
        this.sinif = sinif;
    }

    public String getad() {
        return ad;
    }

    public String getSoyad() {
        return soyad;
    }

    public String getAdSoyad() {
        return ad + soyad;
    }

    public int getNumara() {
        return numara;
    }

    public String getbolum() {
        return bolum;
    }

    public String getfakultebilgi() {
        return fakulte;
    }

    public int getgirisYili() {
        return girisYili;
    }

    public int getsinif() {
        return sinif;
    }
}
